rt_gene/CMakeLists.txt

```diff
@@ -13,13 +13,38 @@ find_package(catkin REQUIRED COMPONENTS
   tf
   tf_conversions
   uvc_camera
+  dynamic_reconfigure
 )
 
 ## System dependencies are found with CMake's conventions
 # find_package(Boost REQUIRED COMPONENTS system)
 
 catkin_python_setup()
 
+################################################
```

Comment:
Can we get rid of those comments? Glad to see you followed the instructions though 👍

Reply:
Agreed